lib/debug/chrome_debugger.dart

```diff
@@ -778,6 +816,15 @@ class ChromeThis extends ChromeDebugBaseVariable {
       : super(connection, frame);
 }
 
+class ChromeEval extends ChromeDebugBaseVariable {
+  final RemoteObject object;
```

Comment:
I generally sort by ctor param order, so, this would be `name` declared first, then `object`.

Reply:
Done